bootstrap.bat

```diff
@@ -51,6 +51,8 @@ IF "%1"=="vc12" SET TOOLSET=msvc : 12.0
 IF "%1"=="vc14" SET TOOLSET=msvc : 14.0
 IF "%1"=="vc141" SET TOOLSET=msvc : 14.1
 
+ECHO.
+ECHO Generating Boost.Build configuration in project-config.jam for %TOOLSET%..."
 ECHO import option ; > project-config.jam
```

Comment:
It would perhaps be nice to have the same preamble here as on unix:

```
 # Boost.Build Configuration
 # Automatically generated by bootstrap.bat
```

Ideally the file would be the same for both platforms.

Reply:
Good point. Fixed.